spec/services/search/postgres/listing_spec.rb

```diff
@@ -0,0 +1,146 @@
+require "rails_helper"
+
+RSpec.describe Search::Postgres::Listing, type: :service do
+  let(:listing) { create(:listing) }
+
+  it "defines necessary constants", :aggregate_failures do
+    constants = [
+      described_class::ATTRIBUTES,
+      described_class::DEFAULT_PER_PAGE,
+      described_class::MAX_PER_PAGE,
+    ]
+
+    constants.each do |constant|
+      expect(constant).not_to be_nil
+    end
+  end
```

Comment:
Nope. Would you like me to remove these specs and make them private?

Reply:
I'm not sure about making them private (I haven't noticed us doing that a whole lot), but it does seem like testing them means we're specifying something that isn't intended to be part of the class interface.

It could just be that how I think about specs is different from how others do, so I'll lay that out real quick so you can see where the question comes from and that might help us figure out what it makes sense to do:

- I consider specs to be an example of how to use an object or class
  - Anything specified in a test could reasonably be expected to be used from anywhere in the application
- Testing implementation details means that any time you refactor you have to change your tests
  - The whole point of refactoring is that the interface doesn't change
  - I don't know what the benefit of testing implementation details is generally and whether it's worth the cost of rewriting the tests when those implementation details change
- I don't consider all tests used in development to be worthwhile as regression tests
  - Some specs are awesome for helping you flesh out the object or method you're working on, but absolutely useless for regression 😄 
    - For example, if it can never fail in a useful way, like `expect(Foo).to belong_to :bar`

Reply:
Okay - removed the specs and made them private (we do it in about 30 other spots).